packages/components/organisms/f-offers/CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v1.0.0-beta.0
+------------------------------
+*August 26, 2021*
+
+### Updated
```

Comment:
Don't think `updated` is part of the changelog `types`.

Reply:
Another one for the follow-up PR, @ashleynolan!

Reply:
I'm going to blame @dandel10n for this one, as she's away on holiday and so isn't here to prove otherwise 😉